packages/design-system/src/components/HelpDrawer/HelpDrawerToggle.jsx

```diff
@@ -37,7 +37,7 @@ HelpDrawerToggle.propTypes = {
   children: PropTypes.node.isRequired,
   /** Additional classes for the toggle button anchor element */
   className: PropTypes.string,
-  /** Add display inline or block to parent span */
+  /** Adds display inline or block style to the HelpDrawerToggle */
```

Comment:
Personally I prefer:
> Adds `display: inline` to the `HelpDrawerToggle`

It doesnt actually add `block` style as thats the default.